syntax = "proto3";

package product_store;

service ProductStoreService{

    rpc CreateProductStore(CreateProductStoreRequest) returns (ProductStore);

}

message ProductStore{
    int32 id = 1;
    int32 store_id = 2;
    int32 product_id = 3;
    int32 stock = 4;
    bool enabled = 5;
}

message CreateProductStoreRequest {
    string store_code = 1;   // Código único de la tienda
    string product_code = 2;  // Código único del producto
    int32 stock = 3;
    bool enabled = 4;
}